### PR TITLE
Issue #1880: Fix remediation for grub2_enable_fips_mode

### PR DESCRIPTION
**Pull request by jan-cerny**
The remediation script missed adding fips=1 parameter
to /etc/default/grub and installing dracut-fips package

### CODE REVIEW

File: RHEL/7/templates/static/bash/grub2_enable_fips_mode.sh [hunk 1]
```diff
@@ -9,6 +9,8 @@ fi
 
 prelink -u -a
 
+yum -y install dracut-fips
```
Comment (redhatrises) [inline RHEL/7/templates/static/bash/grub2_enable_fips_mode.sh]: @jan-cerny please use `package_command install` for this.

Comment (jan-cerny) [inline RHEL/7/templates/static/bash/grub2_enable_fips_mode.sh]: @redhatrises OK

**Comment by jan-cerny** [comment]
I have replaced `yum install` by `package_command install` and I have verified the remediation on RHEL7 again.

**Comment by mpreisler** [comment]
@openscap-jenkins retest this please

**Comment by mpreisler** [comment]
ACK

**Pull request merged**